osbenchmark/test_execution_orchestrator.py

```diff
@@ -185,7 +185,16 @@ def setup(self, sources=False):
         # to derive it ourselves. For source builds we always assume "master"
         if not sources and not self.cfg.exists("builder", "distribution.version"):
             distribution_version = builder.cluster_distribution_version(self.cfg)
-            self.logger.info("Automatically derived distribution version [%s]", distribution_version)
+            if distribution_version == 'oss':
```

Comment:
Is the distribution version when curling against a serverless collection `oss` or `aoss`? Asking because I see the command line specifies `aoss` and am wondering what's the difference

Reply:
Unfortunately curling against root url in serverless results in 404, I can change this to `aoss` for consistency.